templates/common/_base/files/extensions-yum-repo.yaml

```diff
@@ -0,0 +1,11 @@
+{{if .BaseOperatingSystemExtensionsContainer }}
+mode: 0600
+path: "/etc/yum.repos.d/rhel-coreos-8-extensions.repo"
```

Comment:
It'd be a bit of a scramble but we could also try to add the long-overdue `/run/yum.repos.d`.  That'd have different tradeoffs.